manim/constants.py

```diff
@@ -148,3 +148,11 @@ class MyText(Text):
                   if name.endswith("_C")})
 PALETTE = list(COLOR_MAP.values())
 locals().update(COLOR_MAP)
+VERBOSE_CHOICES = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+VERBOSE_FFMPEG_MAP = {
+    "DEBUG": 16,
+    "INFO": 16,
+    "WARNING": 16,
+    "ERROR": 16,
+    "CRITICAL": 8,
```

Comment:
**Question** : Is it normal that all four have the same value?

Reply:
The previous option was by default 16 which corresponds to only showing errors in ffmpeg. I think it makes sense in almost all cases because ffmpeg can be rather verbose.

Reply:
I'm sorry but I don't understand, is there logging level in FFmpeg ? 
Because, I thought that there were those levels `choices=[-8, 0, 8, 16, 24, 32, 40, 48, 56],` available. I'm just wondering, then, why keeping the same value for every level. (I'm not requesting any chang as you said it was ok, it's a legit question).

Reply:
I considered separated loglevel for manim python logger and the loglevel passed to FFmpeg.
I think a manim user won't usually want the output from FFmpeg unless an error happened.
For example, if a manim user has an issue or a developer is testing a new feature, he/she may place logger.debug() in python to output some values. The command that would be run to see these messages would be `manim -v DEBUG` in order to debug an issue. However, it is unlikely the issue would be with FFmpeg so I don't think it is useful to have the debug messages of FFmpeg that can be rather heavy. This is why I kept even for a DEBUG level in manim, an error loglevel for FFmpeg.
I still would assume some users would like to set the loglevel for FFmpeg.
This can be done with a command like `manim -v DEBUG --ffmpeg_loglevel debug`.
The general verbose level DEBUG would set the loglevel of FFmpeg to error due to the `VERBOSE_FFMPEG_MAP` but this is overridden in the shown command by the option `--fmpeg_loglevel`.
I hope it makes it a bit clearer.

Reply:
I get it now. Thank you !